src/Tribe/Main.php

```diff
@@ -1756,12 +1757,51 @@ public function addVenueAndOrganizerEditor() {
 		}
 
 		/**
+		 * Display a helper for the user, about the location and microdata for rich snippets
+		 * @param int $postId the event ID to see if the helper is needed
+		 */
+		public function display_rich_snippets_helper( $post_id ) {
+			// Avoid showing this message if we are on the Front End
+			if ( ! is_admin() ) {
+				return;
+			}
+
+			$venue_id = get_post_meta( $post_id, '_EventVenueID', true );
+			if (
+				( ! $post_id || get_post_status( $post_id ) == 'auto-draft' ) &&
+				! $venue_id &&
```

Comment:
On both of these it doesn't really matter, it's deprecated code. It will be removed in 2 versions.